### PR TITLE
added friends click through and spinners for loading

### PR DESCRIPTION
**Pull request by calhong**
image
Also fixed friends component on the side bar and changed title to white:

image

### CODE REVIEW

**Comment by calhong** [comment]
going to sneak in a small settings change:

image

to 

image

File: src/pages/FriendsInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    console.log("fetching songs");
+    setSongs([]);
+    fetchSongs();
+  }, [fetchSongs]);
+
+  return (
+    <React.Fragment>
+      <header className="App-container">
+        <PageHeader title={`Look at ${username}'s Top Songs!`} />
+        <p>{username}'s most listened to songs:</p>
```
Comment (vandyliu) [inline src/pages/FriendsInfo.jsx:32]: These kinda say the same thing. Maybe we can remove the paragraph or change something else

**Comment by vandyliu** [comment]
Looks good
Also not sure why but my friends hover thing is kinda weird
image

File: src/pages/FriendsInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { getSongs } from "../services";
+import PageHeader from "../components/PageHeader";
+import SongList from "../components/SongList";
+import { connect } from "react-redux";
+import styled, { keyframes } from "styled-components";
```
Comment (vandyliu) [inline src/pages/FriendsInfo.jsx:6]: Can remove if not used

File: src/pages/FriendsInfo.jsx (hunk 1 above)
Comment (calhong) [reply src/pages/FriendsInfo.jsx:31]: It looks really really empty if we remove either one, maybe we can decide nearer the end to see if we can find copy that works for it, or if we just replace it with something else entirely

**Comment by calhong** [comment]
> > > Looks good
> > > Also not sure why but my friends hover thing is kinda weird
> > > image
> > > Hover works properly on `main` though
> > 
> > 
> > @vandyliu Huh... Could you hop on a call with me later today to show me how to replicate this? I tried it on mobile and desktop and I couldn't break it
> 
> If you zoom in, or zoom out, I think it starts to mess up

oh, got it. Should be fixed now.

**Review by vandyliu — APPROVED** [review]
LGTM

**Pull request merged**